tests/lib/eslint.js

```diff
@@ -1741,10 +1741,14 @@ describe("eslint", function() {
             var config = { rules: { "no-alert": 1} };
 
             var messages = eslint.verify(code, config, filename);
-            assert.equal(messages.length, 1);
-            assert.equal(messages[0].ruleId, "no-alert");
-            assert.equal(messages[0].message, "Unexpected alert.");
-            assert.include(messages[0].nodeType, "CallExpression");
+            assert.equal(messages.length, 2);
+
+            // The incorrectly formatted comment triggered a warning
+            assert.equal(messages[0].severity, 2);
```

Comment:
You should check the error message and location to be sure it's the one you're expecting